.travis.yml

```diff
@@ -12,6 +12,7 @@ install:
   - conda create --yes -n env_name python=$PYTHON_VERSION pip numpy scipy nose flake8
   - source activate env_name
   - pip install coveralls
+  - pip install https://github.com/biocore/burrito-fillings/archive/master.zip
```

Comment:
@antgonza is there a reason why mustached-octo-ironman in QIITA [here](https://github.com/biocore/qiita/blob/master/.travis.yml) is being installed via the .travis.yml rather than in setup.py? I was having issues adding burrito-fillings as a dev version in setup.py with the most recent error message being [this](https://travis-ci.org/biocore/deblur/builds/55858598) for this [PR](https://github.com/ekopylova/deblur/commit/e38b4d45d04c2bdf664514c662ea608781113e3e), but everything worked when I applied the solution used in QIITA.